src/pages/ResearcherProfile.js

```diff
@@ -385,400 +304,185 @@ export default function ResearcherProfile(props) {
+                div({ className: 'col-xs-12 no-padding' }, [
+                  div({ className: 'row fsi-row-lg-level fsi-row-md-level no-margin' }, [
+                    eRACommons({
+                      className: 'col-md-4 col-sm-6 col-xs-12',
+                      destination: 'profile',
+                      onNihStatusUpdate: () => {},
+                      location: props.location,
+                      header: false,
+                    }),
                   ])
                 ])
               ]),
 
-              div({ className: 'col-xs-12 no-padding' }, [
-                div({ className: 'row fsi-row-lg-level fsi-row-md-level no-margin' }, [
-                  div({ className: 'col-xs-6' }, [
-                    label({ id: 'lbl_profileAddress1', className: 'control-label' }, ['Street Address 1*']),
-                    input({
-                      id: 'profileAddress1',
-                      name: 'address1',
-                      type: 'text',
-                      className: 'form-control',
-                      defaultValue: profile.address1,
-                      onBlur: handleChange
-                    })
+              div({ className: 'row no-margin'}, [
+                div({ className: 'col-xs-12', style: { 'marginTop': '20px' } }, [
+                  label({
+                    id: 'lbl_profileInstitution',
+                    className: 'control-label',
+                    style: headerStyle,
+                  }, [
+                    'Institution* ',
                   ]),
-                  div({ className: 'col-xs-6' }, [
-                    label({ id: 'lbl_profileAddress2', className: 'control-label' }, ['Street Address 2 ', span({ className: 'italic' }, ['(optional)'])]),
-                    input({
-                      id: 'profileAddress2',
-                      name: 'address2',
-                      type: 'text',
-                      className: 'form-control',
-                      defaultValue: profile.address2,
-                      onBlur: handleChange
-                    })
-                  ])
+                  p({}, ['Please select an institution or enter your institution name if not available from the dropdown.']),
+                  generateInstitutionSelectionDisplay()
                 ])
               ]),
 
-              div({ className: 'col-xs-12 no-padding' }, [
-                div({ className: 'row fsi-row-lg-level fsi-row-md-level no-margin' }, [
-                  div({ className: 'col-xs-6' }, [
-                    label({ id: 'lbl_profileCity', className: 'control-label' }, ['City*']),
-                    input({
-                      id: 'profileCity',
-                      name: 'city',
-                      type: 'text',
-                      className: 'form-control',
-                      defaultValue: profile.city,
-                      onBlur: handleChange
-                    })
+              div({ className: 'row no-margin'}, [
```

Comment:
https://getbootstrap.com/docs/4.0/layout/grid/

These changes here are a lot of spacing changes only - There was a css issue that was being caused by all of these not being in the proper bootstrap format, which should be a class with "row" that contains any classes with "col-[size]-[num]". Without the "row" container, all fo the objects shared the same container, and you couldnt set the background color of one full-width col without affecting multiple at the same time.